pkg/kapp/util/time.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package util
+
+import (
+	"fmt"
+	"time"
+)
+
+func ParseTime(input string, formats []string) (time.Time, error) {
+	for _, format := range formats {
+		t, err := time.Parse(format, input)
+		if err == nil {
```

Comment:
Are we sure errors are only thrown when there is an improper format?
(I think it is, but might be worth verifying)

Reply:
Yeah that I have verified it will only return an error when none of the formats matches.